lib/daysOfTheWeek.js

```diff
@@ -9,3 +9,11 @@ module.exports.getDayName = (date) => {
 
   return ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'][passedDate.getDay()];
 };
+
+module.exports.next = (dow) => {
+  const dows = ['Sunday','Monday','Tuesday',  'Wednesday','Thursday', "Friday",'Saturday'];
+  let days = (dows.indexOf(dow)-(new Date()).getDay());
+  if (days < 0) days+=7;
```

Comment:
This is odd!  What's this for?  Also, the project style guide says that `if` statements should always include brackets (`{ ... }`), along with a couple of rules.  This line is not in compliance.